src/fmi/fmi_import/fmiInfo.h

```diff
@@ -114,10 +113,10 @@ class fmiVariableSet
 	fmiVariableSet();
 	fmiVariableSet(fmi2ValueReference newvr);
 	fmiVariableSet(const fmiVariableSet &vset);
-	fmiVariableSet(fmiVariableSet &&vset) noexcept;
+	fmiVariableSet(fmiVariableSet &&vset);
```

Comment:
I think this could be noexcept in the declaration,  but then you wouldn't use the noexcept in the instantiation